ui/app/clinical/common/views/visitTabPrint.html

```diff
@@ -7,8 +7,8 @@
         <div class="print-header">
             <h1>
                 <img ng-if="visitTabConfig.getPrintConfigForCurrentTab().logo" class="logo" ng-src="{{visitTabConfig.getPrintConfigForCurrentTab().logo}}" alt="logo">
-                {{visitTabConfig.getPrintConfigForCurrentTab().title}}</h1>
-            <h2>{{visitTabConfig.getPrintConfigForCurrentTab().header}}</h2>
```

Comment:
why remove this?

Reply:
We can print Tab Title. In config we should set title to "Patient Visit Summary".

Reply:
@gsluthra @angshu Added title as "Patient Visit Summary"